tamanoir-core/pom.xml

```diff
@@ -11,6 +11,11 @@
     <artifactId>tamanoir-core</artifactId>
     <dependencies>
         <dependency>
+            <groupId>com.jaspersoft.data-discovery</groupId>
```

Comment:
@sergiyblazhievsky  
Why 'com.jaspersoft.data-discovery' but not 'com.jaspersoft.datadiscovery'